src/devices/machine/z80scc.cpp

```diff
@@ -79,7 +79,7 @@ DONE (x) (p=partly)         NMOS         CMOS       ESCC      EMSCC
 // printf("TAG %lld %s%s Data:%d\n", machine().firstcpu->total_cycles(), __PRETTY_FUNCTION__, m_owner->tag(), data);
 
 #define VERBOSE 0
-#define LOGPRINT(x) do { if (VERBOSE) logerror x; } while (0)
+#define LOGPRINT(x) { do { if (VERBOSE) logerror x; } while (0); }
```

Comment:
After 30 years of programming it's not invention, rather lazynes from my side, I have a pattern I use when I develop stuff but I'd be more than happy to convert if we agree on something useful. These LOG printouts are part needed developing the device and part needed when using the device in a board driver as I try to hint about misuse etc. So once the device is complete many of the printout can be removed completely and once we get logerror to work onto something that can be copied and paste on all platforms I'd be glad to remove the redefinition to printf

The extra curly brackets is just a precaution in the case where a LOG is at an unbracketed  else clause as the original has no semicolon. I can of course add the semicolon instead.

I have 29 source files submitted so far and most of them uses this scheme, I suggest that I wait until we have an improved and agreed upon LOG print scheme before I convert and then that I do all of them, what do you think?